public/app/core/components/NavBar/Next/NavBarScrollContainer.tsx

```diff
@@ -36,7 +37,12 @@ export const NavBarScrollContainer = ({ children }: Props) => {
   }, []);
 
   return (
-    <CustomScrollbar className={styles.scrollContainer} hideVerticalTrack hideHorizontalTrack>
+    <CustomScrollbar
+      initializeAtBottom={initializeAtBottom}
```

Comment:
which scrollbar should start at the bottom?

Reply:
@torkelo for the bottom navbar section items where the menu direction is reversed. i don't _think_ this can happen yet, but if we add 1 more item to a menu or if we decide to decrease the max height of the menu it could.

here's what it looks like if i decrease the max height of the menu to 200px instead of the 300px we currently have:

![image](https://user-images.githubusercontent.com/20999846/169566369-47381e12-7585-4161-930a-35b188e3178d.png)

Reply:
ok, I see. Think there might be more items added there soon (either from enterprise or plugins adding items there). Can we increase the max height a bit to give us some buffer? But I guess this is ok, just with the next nav change where global nav will only be via mega menu we will not have this problem, so would be good to not have to introduce the new CustomScrollbar feature if we can

Reply:
@torkelo sure, sounds good 👍